mmdet/datasets/pipelines/loading.py

```diff
@@ -32,16 +32,20 @@ class LoadImageFromFile:
         file_client_args (dict): Arguments to instantiate a FileClient.
             See :class:`mmcv.fileio.FileClient` for details.
             Defaults to ``dict(backend='disk')``.
+        image_backend (str): The backend argument for :func:`mmcv.imfrombytes`.
+            Defaults to None.
     """
 
     def __init__(self,
                  to_float32=False,
                  color_type='color',
-                 file_client_args=dict(backend='disk')):
+                 file_client_args=dict(backend='disk'),
+                 image_backend=None):
```

Comment:
We may use `imdecode_backend` to make it consistent with that in MMSeg, see https://github.com/open-mmlab/mmsegmentation/blob/master/mmseg/datasets/pipelines/loading.py#L36

Reply:
@ZwwWayne when i commit my changes, i get this error: `ModuleNotFoundError: No module named 'setuptools'`

Reply:
```bash
    ERROR: Exception:
    Traceback (most recent call last):
      File "/home/.cache/pre-commit/repoi4zcbvri/py_env-python3/lib/python3.8/site-packages/pip/_internal/cli/base_command.py", line 164, in exc_logging_wrapper
        status = run_func(*args)
      File "/home/.cache/pre-commit/repoi4zcbvri/py_env-python3/lib/python3.8/site-packages/pip/_internal/cli/req_command.py", line 205, in wrapper
        return func(self, options, args)
      File "/home/.cache/pre-commit/repoi4zcbvri/py_env-python3/lib/python3.8/site-packages/pip/_internal/commands/install.py", line 338, in run
        requirement_set = resolver.resolve(
      File "/home/.cache/pre-commit/repoi4zcbvri/py_env-python3/lib/python3.8/site-packages/pip/_internal/resolution/resolvelib/resolver.py", line 73, in resolve
        collected = self.factory.collect_root_requirements(root_reqs)
      File "/home/.cache/pre-commit/repoi4zcbvri/py_env-python3/lib/python3.8/site-packages/pip/_internal/resolution/resolvelib/factory.py", line 468, in collect_root_requirements
        req = self._make_requirement_from_install_req(
      File "/home/.cache/pre-commit/repoi4zcbvri/py_env-python3/lib/python3.8/site-packages/pip/_internal/resolution/resolvelib/factory.py", line 430, in _make_requirement_from_install_req
        cand = self._make_candidate_from_link(
      File "/home/.cache/pre-commit/repoi4zcbvri/py_env-python3/lib/python3.8/site-packages/pip/_internal/resolution/resolvelib/factory.py", line 201, in _make_candidate_from_link
        self._link_candidate_cache[link] = LinkCandidate(
      File "/home/.cache/pre-commit/repoi4zcbvri/py_env-python3/lib/python3.8/site-packages/pip/_internal/resolution/resolvelib/candidates.py", line 281, in __init__
        super().__init__(
      File "/home/.cache/pre-commit/repoi4zcbvri/py_env-python3/lib/python3.8/site-packages/pip/_internal/resolution/resolvelib/candidates.py", line 156, in __init__
        self.dist = self._prepare()
      File "/home/.cache/pre-commit/repoi4zcbvri/py_env-python3/lib/python3.8/site-packages/pip/_internal/resolution/resolvelib/candidates.py", line 225, in _prepare
        dist = self._prepare_distribution()
      File "/home/.cache/pre-commit/repoi4zcbvri/py_env-python3/lib/python3.8/site-packages/pip/_internal/resolution/resolvelib/candidates.py", line 292, in _prepare_distribution
        return preparer.prepare_linked_requirement(self._ireq, parallel_builds=True)
      File "/home/.cache/pre-commit/repoi4zcbvri/py_env-python3/lib/python3.8/site-packages/pip/_internal/operations/prepare.py", line 482, in prepare_linked_requirement
        return self._prepare_linked_requirement(req, parallel_builds)
      File "/home/.cache/pre-commit/repoi4zcbvri/py_env-python3/lib/python3.8/site-packages/pip/_internal/operations/prepare.py", line 546, in _prepare_linked_requirement
        dist = _get_prepared_distribution(
      File "/home/.cache/pre-commit/repoi4zcbvri/py_env-python3/lib/python3.8/site-packages/pip/_internal/operations/prepare.py", line 58, in _get_prepared_distribution
        abstract_dist.prepare_distribution_metadata(finder, build_isolation)
      File "/home/.cache/pre-commit/repoi4zcbvri/py_env-python3/lib/python3.8/site-packages/pip/_internal/distributions/sdist.py", line 47, in prepare_distribution_metadata
        self._install_build_reqs(finder)
      File "/home/.cache/pre-commit/repoi4zcbvri/py_env-python3/lib/python3.8/site-packages/pip/_internal/distributions/sdist.py", line 106, in _install_build_reqs
        build_reqs = self._get_build_requires_wheel()
      File "/home/.cache/pre-commit/repoi4zcbvri/py_env-python3/lib/python3.8/site-packages/pip/_internal/distributions/sdist.py", line 83, in _get_build_requires_wheel
        return backend.get_requires_for_build_wheel()
      File "/home/.cache/pre-commit/repoi4zcbvri/py_env-python3/lib/python3.8/site-packages/pip/_vendor/pep517/wrappers.py", line 172, in get_requires_for_build_wheel
        return self._call_hook('get_requires_for_build_wheel', {
      File "/home/.cache/pre-commit/repoi4zcbvri/py_env-python3/lib/python3.8/site-packages/pip/_vendor/pep517/wrappers.py", line 332, in _call_hook
        raise BackendUnavailable(data.get('traceback', ''))
    pip._vendor.pep517.wrappers.BackendUnavailable: Traceback (most recent call last):
      File "/home/.cache/pre-commit/repoi4zcbvri/py_env-python3/lib/python3.8/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 89, in _build_backend
        obj = import_module(mod_path)
      File "/home/miniconda3/envs/mmdet/lib/python3.8/importlib/__init__.py", line 127, in import_module
        return _bootstrap._gcd_import(name[level:], package, level)
      File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
      File "<frozen importlib._bootstrap>", line 991, in _find_and_load
      File "<frozen importlib._bootstrap>", line 961, in _find_and_load_unlocked
      File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
      File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
      File "<frozen importlib._bootstrap>", line 991, in _find_and_load
      File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
      File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
      File "<frozen importlib._bootstrap_external>", line 783, in exec_module
      File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
      File "/tmp/pip-build-env-tgrkimor/overlay/lib/python3.8/site-packages/poetry/core/masonry/__init__.py", line 10, in <module>
        from .builder import Builder
      File "/tmp/pip-build-env-tgrkimor/overlay/lib/python3.8/site-packages/poetry/core/masonry/builder.py", line 7, in <module>
        from .builders.sdist import SdistBuilder
      File "/tmp/pip-build-env-tgrkimor/overlay/lib/python3.8/site-packages/poetry/core/masonry/builders/__init__.py", line 2, in <module>
        from .wheel import WheelBuilder
      File "/tmp/pip-build-env-tgrkimor/overlay/lib/python3.8/site-packages/poetry/core/masonry/builders/wheel.py", line 23, in <module>
        from packaging.tags import sys_tags
      File "/tmp/pip-build-env-tgrkimor/overlay/lib/python3.8/site-packages/poetry/core/_vendor/packaging/tags.py", line 7, in <module>
        import distutils.util
      File "/home/.cache/pre-commit/repoi4zcbvri/py_env-python3/lib/python3.8/site-packages/_distutils_hack/__init__.py", line 92, in create_module
        return importlib.import_module('setuptools._distutils')
      File "/home/miniconda3/envs/mmdet/lib/python3.8/importlib/__init__.py", line 127, in import_module
        return _bootstrap._gcd_import(name[level:], package, level)
    ModuleNotFoundError: No module named 'setuptools'
```

Reply:
you may re-install the environment and try again?